velox/functions/sparksql/tests/ComparisonsTest.cpp

```diff
@@ -73,10 +73,46 @@ class ComparisonsTest : public SparkFunctionBaseTest {
     auto actual = evaluate<SimpleVector<bool>>(
         fmt::format("{}(c0, c1)", functionName), makeRowVector(input));
     facebook::velox::test::assertEqualVectors(expectedResult, actual);
-  };
+  }
+
+  RowVectorPtr constructRowVector(
+      const std::optional<int64_t>& row,
+      const bool rowIsNull = false) {
+    if (rowIsNull) {
+      return makeRowVector(
+          {makeNullableFlatVector<int64_t>({row})}, nullEvery(1));
```

Comment:
nit: `makeNullableFlatVector<int64_t>({row})` could be reused for the two branches.